front-end/index.html

```diff
@@ -6,10 +6,34 @@
   <script type="text/javascript" src="global.js"></script>
 </head>
 <body>
-
 <h1>Quiz Game</h1>
+<div id="scoreCountNotice"></div> 
+</div>
+<form onsubmit="determineScore(event)">
+<div id="gameEnded" style="display: none">GAME OVER.</div> 
+  <div id="questions">
+
+    <ul id="question" class="question" style="display: none">
+      <p id="questionQuestion"> WORDS WILL GO HERE AND STUFF </p>
+      <li><input type="radio" name="questionChoices" > SOME WORDS WILL GO HERE</li>
+      <li><input type="radio" name="questionChoices" > SOME WORDS WILL GO HERE</li>
+      <li><input type="radio" name="questionChoices" > SOME WORDS WILL GO HERE</li>
+      <li><input type="radio" name="questionChoices" > SOME WORDS WILL GO HERE</li>
+      <li><input type="radio" name="questionChoices" > SOME WORDS WILL GO HERE</li>
 
-<button id="begin_button">Click me to begin the game.</button>
+      <p id="questionActualAnswer" style="display: none" > WORDS WILL GO HERE AND STUFF </p>
+    </ul>
 
+    <p id="correct" style="display: none"> CORRECT </p>
+    <p id="wrong" style="display: none"> WRONG </p>
+
+    <input id="submitButton" style="display: none" type="submit" value="Submit">
+    </form>
+    <button type="button" id="nextButton" style="display: none"> Next Question</button>
+  </div>
+  <button onclick="return getQuestions()" type="button" id="startButton">Click Here to Start the Quiz</button>
```

Comment:
Add event listeners via JavaScript please. This usage of `onclick` in HTML is not a good practice (It's inherently un-DRY and violates SRP).